Amol-Wankhede.md

```diff
@@ -0,0 +1,6 @@
+## Amol Wankhede - Hometown
+
+## My hometown is Nagpur in Maharashtra State (India)
+
+## The temperature here during winter is 10-20 Degrees
+## During Summer its 40-47 Degrees
```

Comment:
@Amol-Wankhede It seems to be pretty hot during summer time.  Any pools or water activities around?